Large-DiT-T2I/README.md

```diff
@@ -27,8 +27,8 @@ For installation instructions, please refer to the [Large-DiT installation guide
 Once the necessary checkpoints are prepared, you can initiate a demo on your local machine using Gradio. Run the following command to get started:
 
 ```bash
-# You can change the --ckpt to different resolution models
-python -u demo.py --ckpt /path/to/trained/ckpt
+# You can change the --ckpt to different resolution models and set --resolution to corresponding model resolution
+python -u demo.py --ckpt /path/to/trained/ckpt --resolution 256
```

Comment:
I think that automatically detecting the resolution from training args will be better